docs/jwst/user_documentation/running_pipeline_python.rst

```diff
@@ -529,3 +529,196 @@ individual step parameter must be set when using this method, or else the coded
 defaults will be used, which may be inappropriate for the dataset being processed.
 
 See :ref:`call_examples` for more information.
+
+
+.. _multiprocessing:
+
+Multiprocessing
+===============
+
```

Comment:
I would start with the pipeline steps where multiprocessing is supported, and an overview of the three scenarios below.  Something like, "Multiprocessing is supported to speed up certain computationally-intensive steps in the pipeline, including X,Y,Z.  The examples below show how multiprocessing can be enabled for these steps, as well as how to set up multiprocessing to simultaneously run the entire pipeline on multiple observations"

Reply:
Thanks, this is a good suggestion.